README.md

```diff
@@ -9,37 +9,68 @@ logstar
 
 It uses winston on the server and `console.log` on the client. It supports multiple arguments. If you have LOGGLY environment variables set, it will auto-configure the winston loggly transport.
 
-#### API
-
-* `Logstar.debug`
-* `Logstar.info`
-* `Logstar.warn`: Prioritize taking action.
-* `Logstar.error`: Take action soon.
-* `Logstar.fatal`: Take action IMMEDIATELY!!!!
-* [SERVER] `Logstar.tags`: Include these tags.
-* [SERVER] `Logstar.winstonInstance`
-
-Example:
-```js
-  import { debug, info, fatal, tags } from 'logstar';
-  tags('system', 'integration');
-  debug('My mood', ['need', 'more', 'coffee']);
-  info({ temperature: -10 });
-  fatal('Sound the alarms', { weather: 'hail' }, new Error('Too cold'));
-  info('more', 'logs');
-  tags();
+## Usage
+```javascript
+import * as Logstar from 'logstar';
+
+Logstar.debug('foo');
+Logstar.info('bar');
+Logstar.fatal('error', new Error('error'));
+```
+
+## API
+
+### `class Logger`
+```javascript
+import { Logger } from 'logstar';
+```
+
+#### `constructor(options = {})`
+Configure loggly here. Optionally provide `meta` to be attached to each log request.
+
+```javascript
+{
+  logLevel: 'info',
+  loggly: {
+    token: '',
+    subdomain: '',
+    tags: '',
+  },
+  meta: {}
+}
+```
+#### `debug(...args)`
+#### `debugf(format, ...args)`
+#### `info(...args)`
+#### `infof(format, ...args)`
+#### `warn(...args)`
+#### `warnf(format, ...args)`
+#### `error(...args)`
+#### `errorf(format, ...args)`
+#### `fatal(...args)`
+#### `fatalf(format, ...args)`
+
+### `transactionLogger`
+Get a logger that adds a `transaction_id` property to the meta. Useful for tracking related requests.
```

Comment:
Yeah exactly. I'd like to instantiate a logger using `getTransactionLogger` on each request and wrap the request in an object that can do like `this.log`. So this is to help us get towards that.